source/fundamentals/gridfs.txt

```diff
@@ -29,7 +29,7 @@ the file storage.
 
 You should use GridFS if the size of your file exceeds the BSON-document
 size limit of 16 megabytes. For more detailed information on whether GridFS is
-suitable for your use case, see the :manual:`GridFS server manual page </core/gridfs>`.
+suitable for your use case, see the :manual:`GridFS Server manual page </core/gridfs>`.
```

Comment:
Suggestion:
I think grouping two proper nouns together could be misinterpreted as if it was a single product name "GridFS Server". I think separating it from the link could be a viable way to clarify this.

```suggestion
suitable for your use case, see the :manual:`GridFS </core/gridfs>` Server manual page.
```

Alternatively:

```
:manual:`GridFS </core/gridfs/>` in the Server manual.